pages/reference-files/[id].js

```diff
@@ -99,6 +123,10 @@ ReferenceFile.propTypes = {
   documents: PropTypes.array,
   // The file is derived from
   derivedFrom: PropTypes.array,
+  // Filesets derived from files belong to
+  derivedFromFileSets: PropTypes.arrayOf(PropTypes.object).isRequired,
+  // Set of documents for file specifications
+  fileFormatSpecifications: PropTypes.array,
```

Comment:
You can add `.isRequired`. If no documents exist, this still receives an empty array, which Javascript and React still consider as something rather than nothing.

Reply:
Changed